scripts/acceptance-tests.bats

```diff
@@ -183,11 +183,11 @@ teardown() {
   vault delete openldap/config
 
   # Remove any roles that were created so they don't bleed over to other tests
-  output=$(vault list -format=json openldap/role || true) # "or true" so it doesn't show an error if there are no roles
+  output=$(vault list -format=json openldap/roles || true) # "or true" so it doesn't show an error if there are no roles
```

Comment:
You'd have to revert these paths as well 😄

Reply:
![https://media.giphy.com/media/xT5LMWJSXHRbSusYve/giphy.gif](https://media.giphy.com/media/xT5LMWJSXHRbSusYve/giphy.gif)